packages/boilerplate-generator/boilerplate-generator.js

```diff
@@ -88,19 +79,18 @@ Boilerplate.prototype._generateBoilerplateFromManifestAndSource =
           readUtf8FileSync(pathMapper(item.path));
       }
     });
-    var boilerplateRenderCode = SpacebarsCompiler.compile(
-      boilerplateSource, { isBody: true });
-
-    // Note that we are actually depending on eval's local environment capture
-    // so that UI and HTML are visible to the eval'd code.
-    // XXX the template we are evaluating relies on the fact that UI is globally
-      // available.
-    global.UI = UI;
-    self.func = eval(boilerplateRenderCode);
-    self.baseData = boilerplateBaseData;
+    this.baseData = boilerplateBaseData;
+  }
 };
 
-var _getTemplate = _.memoize(function (arch) {
-  var filename = 'boilerplate_' + arch + '.html';
-  return Assets.getText(filename);
+// Returns a template function that, when called, produces the boilerplate
+// html as a string.
+const _getTemplate = _.memoize(arch => {
```

Comment:
I don't think the memoize is necessary anymore now that the function isn't doing disk IO.